README.md

```diff
@@ -1,2 +1,171 @@
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- Use semantic HTML tags.
+- Apply best practices in HTML code.
+- Apply JavaScript best practices and language style guides in code.
+- Use JavaScript to manipulate DOM elements.
+- Use JavaScript events.
+- Use objects to store and access data.
+- Communicate technical concepts to other technical people.
+- Use callbacks and promises.
+- Use proper ES6 syntax.
+- Use ES6 modules to write modular JavaScript.
+- Use webpack to bundle JavaScript.
+- Webpack
+- NPM
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, - - - accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add_tasks]**
+- **[Remove_tasks]**
+- **[ave_in_local_storage]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- Use `git clone git@github.com:mrkamin/Leaderboard.git` to clone the repository using the HTTPS link into your local machine
+
+### Setup
+
+- `cd` into your local repository
+
+### Install
+
+- `npm install` - Installs the necessary dependencies
+
+### Usage
+
+- Open the index file and enjoy
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Mohammad Rafi Amin**
+
+- GitHub: [@githubhandle](https://github.com/mrkamin)
+- Twitter: [@twitterhandle](https://twitter.com/Mohamma63974237)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammad-rafi-amin-63b4319b/)
+
+👤 **Author 2**
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add unique ID for each tasks]**
+- [ ] **[Add tasks to the taks list]**
+- [ ] **[edit tasks in the tasks list]**
+- [ ] **[delete tasks in the tasks list]**
+- [ ] **[delete all tasks in the tasks list]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
```

Comment:
- [ ] **[OPTIONAL]:** It would be great also that you include the issues page for your repo, this way, people can contribute to your project by informing about bugs, or requesting additional features! 🤓 🛠️